### ISSUE TITLE
Icons for display options and new notes are changing positions

### ISSUE THREAD

**Issue by xthrk** [standardnotes/forum#2442]
**Describe the bug**
See screenshots below. I open https://app.standardnotes.com/ on the same device in the same browser. One time the icons are on the right, the next time on the top left.

**To Reproduce**
I was not able to reproduce this. Once the icons are to the right of the headline, once above the headline to the left.

**Expected behavior**

**Screenshots**
Icons on the right:
![grafik](https://user-images.githubusercontent.com/116081825/217138602-da0d2044-b91e-4ee0-b648-9019db43fe1d.png)
Icons top left:
![grafik](https://user-images.githubusercontent.com/116081825/217138768-d2af86bc-751a-4782-b4fb-330ced4f28c6.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox 109
 - Version: v3.147.0

**Comment by effieeee** [comment]
Hi @xthrk, is this still the case on Firefox? I'm afraid I wasn't able to reproduce it there either

**Comment by xthrk** [comment]
It is still present. Current Versions: Firefox 109.0.1 and app.standardnotes.com v3.149.0

It never happend in the Standard Notes Desktop App on the same device.

Edit: Added a Screenshot (after signed out, deleted the cache and restart Firefox):
![grafik](https://user-images.githubusercontent.com/116081825/217918753-f62cd81c-a427-46ed-95e9-14f4620b3f90.png)

I have also tested changing the screen resolution and scaling, but without success.
